src/main.rs

```diff
@@ -287,19 +296,16 @@ const APP: () = {
 
     #[task(capacity = 4, priority = 2, resources = [DEBUG_UART, I2C, ACCEL, UBX, GPS_TX, GPS_EN, DELAY])]
     fn gps_event(event: GpsEvent) {
-
+        static mut ACTIVE: bool = false;
         let mut ubx = resources.UBX;
 
-        match event {
-            GpsEvent::AccelInt => {
-                let int_status = resources.ACCEL.get_int_status(resources.I2C).unwrap();
+        let int_status = resources.ACCEL.get_int_status(resources.I2C).unwrap();
+        write!(resources.DEBUG_UART, "INT STAT: {}\r\n", int_status);
 
-                if int_status == 0 {
+        match event {
+            GpsEvent::AccelInt1 => {
+                if !*ACTIVE {
```

Comment:
Might be cleaner to have these as a field in the Ubx struct. We could also think about giving ownership of the UBX pins to the UBX struct, so that pin toggling logic and state updates could all be handled by `ubx.deep_sleep` and `ubx.wake`

Reply:
Don't worry about the latter point into later though. Let's get this PR in